packages/terra-clinical-result/src/flowsheetresultcell/FlowsheetResultCell.jsx

```diff
@@ -0,0 +1,225 @@
+          if (hasStandardResult) {
+            if (i === 0) {
+              const resultItem = resultSet[i];
+              if (resultSet[i].hasComment) { singleResultHasComment = true; resultItem.hasComment = false; }
+              if (resultSet[i].isModified) { singleResultIsModified = true; resultItem.isModified = false; }
+              if (resultSet[i].isUnverified) { singleResultIsUnverified = true; resultItem.isUnverified = false; }
+              if (resultSet[i].eventId) resultKeyID = resultSet[i].eventId;
+              else if (resultSet[i].id) resultKeyID = resultSet[i].id;
+              resultsInnerDisplay = (<ClinicalResult key={(`ClinicalResult-${resultKeyID}`)} resultData={resultItem} hideUnit={hideUnit} isTruncated />);
+            } else if (i > 0) {
+              const hasInterpretation = !isEmpty(resultSet[i].interpretation) ? resultSet[i].interpretation : null;
+              additionalResultInterpretations.push(hasInterpretation);
+            }
+          } else if (hasSystolic || hasDiastolic) {
+            if (i === 0) {
+              const resultItem = resultSet[i];
+              if (hasSystolic) {
+                if (resultSet[i].systolic.hasComment) { singleResultHasComment = true; resultItem.systolic.hasComment = false; }
+                if (resultSet[i].systolic.isModified) { singleResultIsModified = true; resultItem.systolic.isModified = false; }
+                if (resultSet[i].systolic.isUnverified) { singleResultIsUnverified = true; resultItem.systolic.isUnverified = false; }
+              }
+              if (hasDiastolic) {
+                if (resultSet[i].diastolic.hasComment) { singleResultHasComment = true; resultItem.diastolic.hasComment = false; }
+                if (resultSet[i].diastolic.isModified) { singleResultIsModified = true; resultItem.diastolic.isModified = false; }
+                if (resultSet[i].diastolic.isUnverified) { singleResultIsUnverified = true; resultItem.diastolic.isUnverified = false; }
+              }
+              if (resultSet[i].id) resultKeyID = resultSet[i].id;
+              else if (hasSystolic && resultSet[i].systolic.eventId) resultKeyID = resultSet[i].systolic.eventId;
+              else if (hasDiastolic && resultSet[i].diastolic.eventId) resultKeyID = resultSet[i].diastolic.eventId;
+              resultsInnerDisplay = (<ClinicalResultBloodPressure key={(`ClinicalResultBloodPressure-${resultKeyID}`)} resultData={resultItem} hideUnit={hideUnit} isTruncated />);
+            } else if (i > 0) {
+              const sysInterpretation = !isEmpty(resultSet[i].systolic.interpretation) ? resultSet[i].systolic.interpretation : null;
+              const diaInterpretation = !isEmpty(resultSet[i].diastolic.interpretation) ? resultSet[i].diastolic.interpretation : null;
+              additionalResultInterpretations.push(sysInterpretation);
+              additionalResultInterpretations.push(diaInterpretation);
```

Comment:
Since sysInterpretation and diaInterpretation could be `null` do we really want to push potential `null` values into additionalResultInterpretations?